### PR TITLE
Streaming images from local dir

### PR DESCRIPTION
**Pull request by niklub**
Now we can point to local directory, e.g.
```
python server.py --input-path local/directory/with/images
```
and we recursively convert all images to labeling tasks

https://github.com/heartexlabs/label-studio/issues/107

### CODE REVIEW

File: backend/server.py [hunk 1]
```diff
@@ -243,6 +243,12 @@ def api_instruction():
     return make_response(c['instruction'], 200)
 
 
+@app.route('/images/<path:filename>')
```
Comment (makseq) [inline backend/server.py]: почему ты только любых картинок сделал? Кажется можно вообще любые файлы так отдавать.

Comment (makseq) [inline backend/server.py]: Maybe we can support any file types?

Comment (niklub) [reply backend/server.py]: Good point. I've included audio types as well checking when filetypes match current config.

Comment (makseq) [reply backend/server.py]: I'd like to rename /images/ to /assets/ or something like and proxy any type of files over this

**Pull request merged**